core/src/main/java/me/deadlight/ezchestshop/data/ShopContainer.java

```diff
@@ -446,7 +447,14 @@ private static boolean ifHasMoney(OfflinePlayer player, double price) {
         }
     }
 
-    private static void getandgive(OfflinePlayer withdraw, double price, OfflinePlayer deposit) {
+    private static void getandgive(OfflinePlayer withdraw, double price, OfflinePlayer deposit, double taxesvalue) {
+        if(Config.taxesshopplayer != 0){
+            double taxes = price*(taxesvalue)/100;
+            price = price*(100-taxesvalue)/100;
+            if(!(Config.leader.equalsIgnoreCase("[]"))){
```

Comment:
When is this value ever going to be `[]`? Server Owners don't know that they should use `[]` to say that this should not go to a player. It would be way better to check for an empty string. Also you do not have any error handling in case that player name isn't found or does not exist. Would recommend to add that.